tests/Jwt/FirebaseParserTest.php

```diff
@@ -65,6 +65,20 @@ public function testParseTokenWithExpiredToken()
         }
     }
 
+    public function testParseTokenWithInvalidToken()
+    {
+        $parser = new FirebaseParser($this->config);
+
+        try {
+            $parser->parseToken(
+                'undefined'
+            );
+            $this->fail('Expected exception was not thrown');
+        } catch (InvalidException $e) {
```

Comment:
@shadowhand It's a stylistic difference, and I don't think it's one that warrants blocking this PR. If the difference bothers you that much, you can always submit a separate PR to handle those changes. :wink: